bootstrap/manifest_template.json

```diff
@@ -67,6 +67,34 @@
     "url": "http://postgres-api.discoverd/ping"
   },
   {
+    "id": "logaggregator",
```

Comment:
It should come after the scheduler. `run-app` is only used to bootstrap hard dependencies of the controller, `deploy-app` should be used after the controller+scheduler are running as it uses the controller API instead of emulating the behavior of the controller/scheduler in bootstrap.